module/Application/config/module.config.php

```diff
@@ -24,21 +25,62 @@
                     ],
                 ],
             ],
-            'application' => [
-                'type'    => Segment::class,
+            'account' => [
+                'type' => Literal::class,
                 'options' => [
-                    'route'    => '/application[/:action]',
+                    'route'    => '/account',
                     'defaults' => [
-                        'controller' => Controller\IndexController::class,
+                        'controller' => Controller\AccountController::class,
                         'action'     => 'index',
                     ],
                 ],
+                'may_terminate' => false,
+                'child_routes' => [
+                    'get' => [
+                        'type' => Method::class,
+                        'options' => [
+                            'verb' => 'get',
+                            'defaults' => [ 'action' => 'index'],
+                        ]
+                    ],
+                    'post' => [
+                        'type' => Method::class,
+                        'options' => [
+                            'verb' => 'post',
+                            'defaults' => [ 'action' => 'post'],
+                        ]
+                    ],
+
+                ],
+            ],
+            'login' => [
+                'type'    => Literal::class,
+                'options' => [
+                    'route'    => '/login',
+                    'defaults' => [
+                        'controller' => Controller\LoginController::class,
+                        'action'     => 'login',
+                    ],
+                ],
             ],
+            'logout' => [
+                'type'    => Literal::class,
+                'options' => [
+                    'route'    => '/logout',
+                    'defaults' => [
+                        'controller' => Controller\LogoutController::class,
```

Comment:
does this need its own controller? can we integrate with login controller and rename?